sql/src/main/java/io/crate/analyze/relations/RelationNormalizer.java

```diff
@@ -138,13 +138,13 @@ private static QuerySpec mergeQuerySpec(@Nullable QuerySpec querySpec1, QuerySpe
 
         return new QuerySpec()
             .outputs(querySpec1.outputs())
-            .where(mergeWhere(querySpec1.where(), querySpec2.where()))
-            .orderBy(mergeOrderBy(querySpec1.orderBy(), querySpec2.orderBy()))
-            .offset(Limits.add(querySpec1.offset(), querySpec2.offset()))
-            .limit(mergeLimit(querySpec1.limit(), querySpec2.limit()))
-            .groupBy(pushGroupBy(querySpec1.groupBy(), querySpec2.groupBy()))
-            .having(pushHaving(querySpec1.having(), querySpec2.having()))
-            .hasAggregates(querySpec1.hasAggregates() || querySpec2.hasAggregates());
+            .where(mergeWhere(querySpec2.where(), querySpec1.where()))
```

Comment:
Should we rename the variables into parentSpec / childSpec or something to make the relation between the two more clear?